pyproject.toml

```diff
@@ -6,3 +6,13 @@ requires = [
     "Cython>=0.29",
     "oldest-supported-numpy",
 ]
+
+[tool.cibuildwheel]
+skip = ["cp36-*", "cp37-*", "pp*"]
+archs = ["auto64"]
+
+[tool.cibuildwheel.linux.environment]
+GDAL_INCLUDE_PATH = "/vcpkg/packages/gdal_x64-linux/include"
+GDAL_LIBRARY_PATH = "/vcpkg/packages/gdal_x64-linux/lib"
```

Comment:
If works similar as on windows, the path might also be something like `/vcpkg/installed/x64-linux/lib`

Reply:
Indeed, that looks to be it.